src/core/Akka/Routing/Broadcast.cs

```diff
@@ -185,60 +145,116 @@ public override RouterConfig WithFallback(RouterConfig routerConfig)
         {
             return OverrideUnsetConfig(routerConfig);
         }
-    }
 
-    /// <summary>
-    /// This class represents a <see cref="Group"/> router that sends messages it receives to all of its routees.
-    /// </summary>
-    public class BroadcastGroup : Group
-    {
-        /// <summary>
-        /// This class represents a surrogate of a <see cref="BroadcastGroup"/> router.
-        /// Its main use is to help during the serialization process.
-        /// </summary>
-        public class BroadcastGroupSurrogate : ISurrogate
+        private RouterConfig OverrideUnsetConfig(RouterConfig other)
```

Comment:
We have this method in each pool router. I can't move it to the base class, as it was before. Because in that case we will have api inconsistency with Scala version